modules/api/src/main/java/org/apache/ignite/table/mapper/Mapper.java

```diff
@@ -26,6 +28,10 @@
  * @param <T> Mapped type.
  */
 public interface Mapper<T> {
+    /** Identity mapper cache. */
+    @SuppressWarnings("rawtypes")
+    static final Map<Class, IdentityMapper> identityMappers = new ConcurrentHashMap<>();
```

Comment:
This is  public API class that is used as on client-side as on server-side.
On client-side, this cache is ok for most use cases, because amount of user domain classes is limited.
But on-server side, user may want to use table views with mappers, but classes may be deployed temporary for compute purposes. 
Thus, cache can be bloated.

Let's move mapper caching to a separate task.

Reply:
Ok, agree.

* Created https://issues.apache.org/jira/browse/IGNITE-16094
* Removed Marshaller and Mapper caching